plugins/upload-assets-to-s3/README.md

```diff
@@ -0,0 +1,79 @@
+# @dotcom-tool-kit/upload-assets-to-s3
+
+Upload files to a configured AWS S3 bucket.
+
+## Installation
+
+Install `@dotcom-tool-kit/upload-assets-to-s3` as a `devDependency` in your app:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/upload-assets-to-s3
+```
+
+Add the plugin to your [Tool Kit configuration](https://github.com/financial-times/dotcom-tool-kit/blob/main/readme.md#configuration):
+
+```yaml
+plugins:
+	- '@dotcom-tool-kit/upload-assets-to-s3'
+```
+
+Install this plugin's tasks:
+
+```sh
+npx dotcom-tool-kit --install
+```
+
+## Options
+| Key | Description | Default value |
+|-|-|-|
+| `accessKeyIdEnvVar` | variable name of the project's aws access key id | no default value - for backwards compatability the plugin falls back to the default value for `accessKeyId` |
+| `secretAccessKeyEnvVar` | variable name of the project's aws secret access key | no default value - for backwards compatability the plugin falls back to the default value for `secretAccessKey` |
+| `accessKeyId` | **DEPRECATED** variable name of the project's aws access key id | 'aws_access_hashed_assets' |
+| `secretAccessKey` | **DEPRECATED** variable name of the project's aws secret access key | 'aws_secret_hashed_assets' |
+| `directory` | the folder in the project whose contents will be uploaded to S3 | 'public' |
+| `reviewBucket` | the development or test S3 bucket | ['ft-next-hashed-assets-preview'] |
+| `prodBucket` | production S3 bucket/s; an array of strings | ['ft-next-hashed-assets-prod', 'ft-next-hashed-assets-prod-us'] |
```

Comment:
**question**: if you do pass an array of string does this mean the same files will be uploaded in the same places, just in different buckets?

Reply:
[Yes](https://github.com/Financial-Times/dotcom-tool-kit/blob/main/plugins/upload-assets-to-s3/src/tasks/upload-assets-to-s3.ts#L56-L67):

```ts
        for (const bucket of bucketByEnv) {
          const params = {
            Bucket: bucket,
            Key: key,
            Body: body,
            ACL: 'public-read',
            ContentType: `${type}; charset=utf-8`,
            ContentEncoding: encoding,
            CacheControl: options.cacheControl
          }
          currentBucket = params.Bucket
          const data = await s3.upload(params).promise()
```

Reply:
I have clarified in the README 😊